public/modules/custom/grants_handler/src/Plugin/WebformHandler/GrantsHandler.php

```diff
@@ -1446,6 +1446,8 @@ public function confirmForm(
 
       // If application uploaded succesfully.
       if ($applicationUploadStatus) {
+        // Erase other messages.
+        $this->messenger()->deleteAll();
```

Comment:
Mä en oo varma onks tää ratkasu. 

Ku toi on tossa ni mä en nää mitään viestejä, en ees tota onnistuneesta tallennuksesta. Vaikka se onki tossa jälkeenpäin.

Näkyks sulla tuo tallennusviesti?

Mä en myöskään näe sitä keltasta viestiä millään kombolla. Eiks tää sen viestin takia laitettu?

Reply:
Mä jotenki epäilen ton keltasen boksin tulleen esim jostain välimuistista tms.

Eikä tuota näy nytkään STAGElla:
![image](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/assets/892920/56fdf918-5a0a-4d61-8bee-79589e8f1b6f)

Reply:
Mä näen onnistuneen viestin ennen kuin sivu uudelleenlataa itsensä onnistuneesti vastaanotetusta (jolloin sen ei pitäisi enää näkyä). Mä en ole keksinyt, miten pääsen deletoimaan yksittäisen viestin jonosta, ja kun tonne sivulle mennään vain ja ainoastaan kun homma on jo mennyt läpi oikein, en ole varma mitä muuta viestejä me haluttaisi siellä näyttää. Jos keksit, miten voi kaivaa viestin, jota me ei itse aseteta, ja poistaa sen näyttämisen, niin olen korvina.

Reply:
Mulla tää ainaki meni just niinku on kuvattuki, toisessa branchissa tuli keltanen loota lähetyksen jälkeen ja kaikki hävis ku vaihtu Vastaanotettuun, ja tässä branchissa ei tullu keltasta lootaa ja muuten toimi samalla tavalla

Reply:
OK, ehkä mä / mun kone on tossa sekoillu jos toimii suvilla oikein